cvat/apps/engine/models.py

```diff
@@ -586,10 +614,8 @@ class Task(TimestampedModel, FileSystemRelatedModel):
     name = SafeCharField(max_length=256)
     mode = models.CharField(max_length=32)
     owner = models.ForeignKey(User, null=True, blank=True,
-        on_delete=models.SET_NULL, related_name="owners")
-    assignee = models.ForeignKey(User, null=True,  blank=True,
-        on_delete=models.SET_NULL, related_name="assignees")
-    assignee_updated_date = models.DateTimeField(null=True, blank=True, default=None)
+        on_delete=models.SET_NULL, related_name="owned_tasks")
```

Comment:
It would be more consistent to use "tasks" here - consider, for example, the equivalent field in `CloudStorage` or `Asset`.

Also, as long as you're updating this field, please add a `related_query_name`.